<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Log</class>
 <widget class="QWidget" name="Log">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log</string>
  </property>
  <widget class="QPlainTextEdit" name="LogText">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>720</height>
    </rect>
   </property>
   <property name="plainText">
    <string>这里是日志文件</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
